scripts/build/Jenkins/Jenkinsfile

```diff
@@ -247,7 +247,7 @@ def CheckoutRepo(boolean disableSubmodules = false) {
                 [$class: 'CheckoutOption', timeout: 60]
             ],
             userRemoteConfigs: scm.userRemoteConfigs
-       ]
+       ]       
```

Comment:
cosmetic: seems this is adding unnecessary spaces

Reply:
fixed 👍